Tests/test_lib_image.py

```diff
@@ -30,3 +60,59 @@ def test_setmode():
         im.im.setmode("L")
     with pytest.raises(ValueError):
         im.im.setmode("RGBABCDE")
+
+
+@pytest.mark.parametrize("mode", mode_names_not_bgr)
+def test_equal(mode):
+    num_img_bytes = len(Image.new(mode, (2, 2)).tobytes())
+    # alternatively, random.randbytes() in Python 3.9
+    data = secrets.token_bytes(num_img_bytes)
```

Comment:
Apart from oss-fuzz, Pillow doesn't use random data in the test suite.

Reply:
Different image modes need different amounts of bytes to create them, so this seemed like the best way to get enough bytes, and then get a second set of bytes that are different.